app/models/concerns/users/info.rb

```diff
@@ -18,11 +18,21 @@ def member_at?(organization)
       organization_user_at(organization).present?
     end
 
-    def organizations_with_admin_access
+    def organizations_with_access
       if super_admin?
-        @organizations_with_admin_access ||= Organization.all
+        @organizations_with_access ||= Organization.all
       else
-        organizations.select { |org| admin_at?(org) }
+        organizations
+      end
+    end
+
+    def organizations_with_permission(permission)
```

Comment:
This is sort of new in this branch. Basically, before this branch, I had `organizations_with_admin_access` to check which organizations you have admin access to... which is fine at first.

Except that lead to a bit of a problem. Take inviting a user as an example... the view and controller and everything would be checking if you have the invite permission in general, or invite permission at the given organization. Then listing the organizations would be this more general method of all organizations you have admin access. This means that accessing the organizations you can invite to isn't quite consistent with the rest of the code... that's fine, but we had talked about potentially fine grained permissions for everyone. If/when we get to that... all code would be super easy to update... *except* this list of organizations. You would have to look for these method calls and then figure out what the intention was for the list of organizations, then create a corresponding method and whatnot.

This led me to create `organizations_with_permission`, which allows the view to list the organizations that you have access to invite to. Now all the code has the same intention revealing names (it's clear you are getting access to all organizations that you have that specific permission for). It also means adding fine grained permissions means updating a small handful of methods for each fine grained permission.

So then the difference is `organizations_with_access` is a general method to list all organizations you have any access to, and if you need organizations for a particular purpose, you would pass the permission needed for that organizations, so in invite users, it would be `:can_invite_user_at?`.

That's the intention of all those changes anyways.